commons/src/main/res/values/styles.xml

```diff
@@ -228,8 +228,9 @@
         <item name="android:buttonBarNeutralButtonStyle">@style/MaterialDialogButtonStyle</item>
     </style>
 
-    <style name="Theme.Transparent" parent="Theme.AppCompat.Light.NoActionBar">
+    <style name="Theme.Transparent" parent="Theme.Material3.DayNight.NoActionBar">
```

Comment:
we arent using DayNight anywhere, do you have any specific reason why isnt it Dark?

Reply:
material widgets like the dialog use the surface color from the parent theme. If we don't use DayNight or define light/dark variants, then it'll stay dark even when using the light theme:

<img src="https://user-images.githubusercontent.com/36371707/185763908-a2d02c7a-2aa6-433f-ba1e-f3ab47c3efa0.png" width="264" />

Reply:
we have control over the dialogs, lets use Dark theme and color them dynamically, just like always

Reply:
[done :+1: ](https://github.com/SimpleMobileTools/Simple-Commons/pull/1486/commits/89edebbc7ba252ec4d8c92b347ee14b261ef399f)

had to at least specify the dialog theme so as to not tinker with dialog creation code ([it will otherwise use theme background](https://github.com/SimpleMobileTools/Simple-Commons/blob/59d56ae16f1afe0cb13c134a42bd12b364ff8be7/commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Context-styling.kt#L23))

Reply:
have you tested it on lower OS versions too? It might crash because of missing material design stuff.

Reply:
I only tested on lollipop (22), 11, and 12, testing on other versions...

Reply:
all good :+1:

Reply:
> It might crash because of missing material design stuff

the "System default" option is hidden on previous versions

Reply:
Theme.Transparent can be used anywhere, it doesnt have to be system default

Reply:
Actually, Material3 is supported on older versions too (except dynamic theming obviously)

<img src="https://user-images.githubusercontent.com/36371707/185780639-a5a46a4a-6baf-4e3b-b69a-a2ae2c016a27.png" width="264" alt="Lollipop (22)"/>   <img src="https://user-images.githubusercontent.com/36371707/185780610-6e87f3b4-b4ee-48aa-9fc9-118bd945db6b.png" width=264 />

> It might crash because of missing material design stuff

Can confirm, it works fine.